runway/module/serverless.py

```diff
@@ -103,20 +102,23 @@ def run_serverless(self, command='deploy'):
                                            env_vars=self.context.env_vars)
             else:
                 LOGGER.warning(
-                    "Skipping serverless %s of %s; no \"package.json\" "
+                    "Skipping 'serverless %s' for %s; no \"package.json\" "
                     "file was found (need a package file specifying "
                     "serverless in devDependencies)",
                     command,
-                    os.path.basename(self.path))
+                    self.name)
         else:
             response['skipped_configs'] = True
             LOGGER.info(
-                "Skipping serverless %s of %s; no config file for "
-                "this stage/region found (looking for one of \"%s\")",
+                "Skipping 'serverless %s' for '%s' in %s; "
+                "no configuration found for this stage/region.",
                 command,
-                os.path.basename(self.path),
-                ', '.join(gen_sls_config_files(self.context.env_name,
-                                               self.context.env_region)))
+                self.name,
+                self.context.env_region)
+            LOGGER.info("Looking for one of the following in root or 'env' folder:")
```

Comment:
If found the list on a single line confusing visually, so broke them out on individual lines.... though lately I'm thinking this might not be valuable.